### ISSUE TITLE
'Cannot read property 'test' of undefined' with Angular CLI 6.2.4 / Angular 6.1.9

### ISSUE THREAD

**Issue by blumk** [froko/ng-essentials#4]
Steps to reproduce:

`ng new sample-app --style scss`

`ng add @froko/ng-essentials --jest --cypress`

```
WARNING: This command may not execute successfully. The package/collection may not support the 'targets' field within 'C:\sample-app\angular.json'. This can be corrected by renaming the following 'targets' fields to 'architect':
1) Line: 16; Column: 7
2) Line: 105; Column: 7

Cannot read property 'test' of undefined
```

**Comment by froko** [comment]
I got this error as well in the past, but cannot reproduce it with the current version. Have you tried to update the global @angular/cli package?

**Comment by froko** [comment]
@blumk: Can you check your angular.json? Is there an "architect" node or something else?

**Comment by blumk** [comment]
@froko I had to replace "target" with "architect". It seems to be a @angular/cli bug fixed with [#12230](https://github.com/angular/angular-cli/issues/12230)

Updating the global @angular/cli packaged fixed the issue.
